doc/en/user/source/security/csp.rst

```diff
@@ -0,0 +1,295 @@
+.. _security_csp:
+
+Content Security Policy
+=======================
+
+The :guilabel:`Content Security Policy` page controls the checks that are performed on GeoServer
+requests that GeoServer will use to set an appropriate Content-Security-Policy (CSP) HTTP
+response header to mitigate cross-site scripting and clickjacking attacks.
+
+.. note::
+    The settings on this page do not affect anything if the CSP header has been disabled using the
+    ``geoserver.csp.shouldSetPolicy`` property or if the ``geoserver.csp.policy`` property is used
+    to set a fixed header value.
+
+.. note::
+    Starting with version 9, Wicket will set its own CSP header with a very strict policy that will
+    overwrite what is set by GeoServer although there is a workaround in place to merge directives
+    set by GeoServer that are either not supported or not set by default by Wicket (particularly
+    ``form-action`` and ``frame-ancestors``). Wicket's CSP is separate from GeoServer's
+    functionality and will otherwise not be affected by GeoServer's configuration.
+
+Default Configuration
+---------------------
+
+The default CSP configuration is intended to support many GeoServer use cases and allow users to
+securely run GeoServer without having to modify the configuration. It may be updated in future
+releases to fix bugs, support new features or enhance security.
+
+The default header value for most GeoServer requests will be (without line breaks):
+
+::
+    base-uri 'self'; form-action 'self'; default-src 'none'; child-src 'self'; connect-src 'self';
+    font-src 'self'; img-src 'self' data:; style-src 'self' 'unsafe-inline'; script-src 'self';,
+    frame-ancestors 'self';
```

Comment:
Not sure what you wanted to get here, but the rendering is weird (happens in the other examples in the pages as well):

![image](https://github.com/geoserver/geoserver/assets/325433/7dd018f4-4ae4-4c48-bbb6-0b57db019a41)

Also, is this comma intended? ``script-src 'self';,``

Reply:
The rendering is fixed now.